TTS/api.py

```diff
@@ -122,8 +122,9 @@ def languages(self):
     def get_models_file_path():
         return Path(__file__).parent / ".models.json"
 
+    @staticmethod
     def list_models(self):
```

Comment:
```suggestion
    def list_models():
```

You also need to sign the CLA for the PR to be merged.